common/_locales/en/messages.json

```diff
@@ -1141,7 +1141,7 @@
 				"description": "Paragraph 2, please translate this text."
 			},
 			"paragraph3": {
-				"content": "Network Boundary Shield default behavior can be globally adjusted by the settings below. Network Boundary Shield can also be completely disabled per domain by the exception list. Domains can be allowed to make cross-network requests via <i>Network Boundary Shield switch</i> in the popup window or manually via the <i>Manage exception list</i> option. Remember that adding a domain to the exception list also allows its subdomains to make cross-network requests. To selectively deactivate Network Boundary Shield, insert the domains to the exception list (excluding \"www,\" but including all other domains, e.g., \".com\").",
+				"content": "Network Boundary Shield default behavior can be globally adjusted by the settings below. Network Boundary Shield can also be completely disabled per domain by the exception list. Domains can be allowed to make cross-network requests via <i>Network Boundary Shield switch</i> in the popup window or manually via the <i>Manage exception list</i> option. Remember that adding a domain to the exception list also allows its subdomains to make cross-network requests. To selectively deactivate Network Boundary Shield, insert the domains to the exception list (excluding \"www.\" but including all other domains, e.g. \".com\").",
```

Comment:
This one should be a comma. However, I am not sure if the correct positionining is after or before the '"' character

Reply:
I meant point in "www." as part of URL, not a punctuation mark, like ".com" later. Added comma back for better understanding.

Reply:
Or you talking about this comma "e.g.,"?

Reply:
I thoght about that and I go for:
"www" without comma or dot,
e.g., (with comma).
I am leaving this way until some native speaker says otherwise.